pep-0618.rst

```diff
@@ -117,9 +115,6 @@ When the built-in ``zip`` is called with the keyword-only argument
 the arguments are exhausted at differing lengths.  This error will
 occur at the point when iteration would normally stop today.
 
-At most one additional item may be consumed from one of the iterators
-when compared to normal ``zip`` usage.
-
 
 Backward Compatibility
```

Comment:
I would have expanded upon this slightly, e.g. pointing out that zip() currently does not take keyword arguments and that the default is strict=False.